aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -152,6 +152,11 @@ public static CreateDbInstanceReadReplicaRequest createDbInstanceReadReplicaRequ
             builder.storageThroughput(model.getStorageThroughput());
             builder.storageType(model.getStorageType());
         }
+
+        if (ResourceModelHelper.isMySQL(model)) {
```

Comment:
This line stands out. I understand this is because of particularities of the engine but we should document this somewhere so that we can revisit this code in the future. Perhaps in the PR description we could add some context of why this is needed.